cmd/project.go

```diff
@@ -105,6 +106,23 @@ var projectCreateCmd = &cobra.Command{
 	},
 }
 
+var projectDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete a project",
+	Long:  "Delete a project",
```

Comment:
What do you plan to delete ? 
- All the k8's/openshift resources added to a namespace or not and selected using k8s label
- The current namespace
What is a project ? Can you improve the description of the command and what will be done please ?

Reply:
@cmoulliard I am changing the long description to "Delete a project and all resources deployed in the project being deleted" 

I am not sure namespace as a term is used by openshift but project is a openshift term AFAIK